### PR TITLE
Changes for NRP 2023-04-01 Release

### PR DESCRIPTION
**Pull request by Joanna-Yang-Art**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by Joanna-Yang-Art** [comment]
Hi @vidai-msft and @Nickcandy Could anyone of you help approve PowerShell PR for NRP 2023-04-01 Release https://github.com/Azure/azure-powershell/pull/22371, we are trying to catch up with the following PowerShell release on 8/1

Attached follow up discussion from last release, should be the same here

image

**Comment by Nickcandy** [comment]
@Joanna-Yang-Art Please recover the .placeholder first and let's see if the CI could pass

**Comment by Joanna-Yang-Art** [comment]
> @Joanna-Yang-Art Please recover the .placeholder first and let's see if the CI could pass

Sure, Aadded back .placeholder.txt

File: src/Network/Network.Management.Sdk/README.md [hunk 1]
```diff
@@ -25,69 +25,69 @@ license-header: MICROSOFT_MIT_NO_VERSION
...
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-02-01/virtualNetworkTap.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-02-01/virtualRouter.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-02-01/virtualWan.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-02-01/vmssNetworkInterface.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-02-01/vmssPublicIpAddress.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-02-01/webapplicationfirewall.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/network/resource-manager/Microsoft.Network/stable/2023-04-01/applicationGateway.json
```
Comment (Nickcandy) [inline src/Network/Network.Management.Sdk/README.md:30]: https://github.com/Azure/azure-rest-api-specs/blob/$(commit)/specification/network/resource-manager/Microsoft.Network/stable/2023-04-01/applicationGateway.json
Please change the url like this.

Comment (Nickcandy) [reply src/Network/Network.Management.Sdk/README.md:30]: If the CI still failed, please fix the test issue. The code freeze would be on next Tuesday. Please fix them ASAP or we would postpone the pr to next release.

**Review by Nickcandy — APPROVED** [review]

**Pull request merged**